src/common/components/multiplayer/Chat.js

```diff
@@ -184,11 +185,13 @@ export default class Chat extends Component {
             height: this.state.optionsVisible ? 'calc(100% - 92px - 144px)' : 'calc(100% - 144px)',
             overflowY: 'scroll'
           }}>
-          {
-            this.mergeMessagesById(messages)
-              .filter(this.filterMessage.bind(this))
-              .map((message, idx) => <ChatMessage key={idx} message={message} idx={idx} />)
-          }
+          <TransitionGroup>
+            {
+              this.mergeMessagesById(messages)
+                .filter(this.filterMessage.bind(this))
```

Comment:
You can avoid the `.bind(this)` if you declare it as an arrow function. Here and the line below.